examples/20_basic/example_regression.py

```diff
@@ -44,5 +44,20 @@
 # Get the Score of the final ensemble
 # ===================================
 
-predictions = automl.predict(X_test)
-print("R2 score:", sklearn.metrics.r2_score(y_test, predictions))
+train_predictions = automl.predict(X_train)
+print("Train R2 score:", sklearn.metrics.r2_score(y_train, train_predictions))
+test_predictions = automl.predict(X_test)
+print("Test R2 score:", sklearn.metrics.r2_score(y_test, test_predictions))
+
+###########################################################################
+# Plot the predictions
+# ===================================
```

Comment:
Do you think it makes sense to add more intuition about this plot? So this is a basic example, so someone might not directly see the use of this plot?

Reply:
Yes, I added more text to explain the plot and the metrics

Reply:
Agree